deploy/galaxy/main.tf

```diff
@@ -42,6 +42,23 @@ resource "google_storage_bucket" "secure" {
   }
 }
 
+resource "google_storage_bucket" "ephemeral" {
+  name = "mitbattlecode-${var.name}-ephemeral"
+
+  location      = var.gcp_region
+  storage_class = "STANDARD"
+  labels        = merge(var.labels, {component="storage"})
+
+  lifecycle_rule {
+    condition {
+      age = 1
```

Comment:
its nice that the zip dies after a day but...hm...
It's a direct link to download right? Users wouldn't be saving the _link_ for any real reason?
If so then yay
(if not then...if they try to go back and get hit w a cryptic error or smth, might be annoying; would like to try to find a way to explain/handle better)

Reply:
they shouldn't save the link- in fact, they don't actually need to see the link ever
browsers just download the file and that's the end of it

Reply:
if they lose the file, too bad, go and download it again